README.md

```diff
@@ -56,9 +56,11 @@ All commands in code font are to be copied and pasted onto command line.
 1. `rbenv init`   
 1. `rbenv local 3.0.2`
 1. Close the CLI tab and open a new one
-1. `cd path/tp/shelter-assist`
+1. `brew install yarn`
```

Comment:
May not hurt to link to https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable just in case brew doesn't work out.